Tests/Source/Data Model/ServiceUserTests.swift

```diff
@@ -22,6 +22,39 @@ import XCTest
 @testable import WireSyncEngine
 
 final class DummyServiceUser: NSObject, ServiceUser {
+    var availability: Availability = .none
+    
+    var teamName: String? = nil
+    
+    var isBlocked: Bool = false
+    
+    var isExpired: Bool = false
+    
+    var isPendingApprovalBySelfUser: Bool = false
+    
+    var isPendingApprovalByOtherUser: Bool = false
+    
+    var isWirelessUser: Bool = false
+    
+    var expiresAfter: TimeInterval = 0
+    
+    var readReceiptsEnabled: Bool = true
+    
+    var richProfile: [UserRichProfileField] = []
+    
+    var canCreateConversation: Bool = true
```

Comment:
Bots cannot create conversations.

```suggestion
    var canCreateConversation: Bool = false
```